### ISSUE TITLE
[Bug]: Adding Media to a post seems to behave erractically (Mobian Trixie/L5)

### ISSUE THREAD

**Issue by kop316** [GeopJr/Tuba#597]
### Describe the bug

I am trying to post on Tuba, and when I try to add media, it does not seem to work. Clicking on it seems to bring up a pop up menu `Paste` more often than not, but if I click on it enough times, it will work.

### Steps To Reproduce

Click on the Pencil Icon (for new post), go to `Media`, click `Add Media`

### Instance Backend

Mastodon

### Operating System

Mobian

### Package

Flatpak

### Troubleshooting information

This happens on the Flatpak and native Debian App.

**Comment by GeopJr** [comment]
I assume this is a mobile device using touch as input, right?

Just to make sure, does it work as expected when you edit your profile? (Go to your Profile, click the pencil icon on the headerbar, on the new window click the pencil icon on your profile picture or cover)

If it does, then it's probably a gesture issue:

There's something wrong with gestures on mobile (a similar bug occurred some versions ago with a swipe-down gesture being activated on touch instead which ended with removing it altogether (#297)).

It's supposed to open the context menu on secondary click or long tap

This is where the gestures get added:

https://github.com/GeopJr/Tuba/blob/3ddccee4aa62b4013f7e2609174ce221d3723982/src/Dialogs/Composer/AttachmentsPage.vala#L88-L129

I can't reproduce it on desktop or qemu so I'm leaving it as is for now.

If it's too annoying, you can remove the gesture by commenting out the following:

https://github.com/GeopJr/Tuba/blob/3ddccee4aa62b4013f7e2609174ce221d3723982/src/Dialogs/Composer/AttachmentsPage.vala#L92

https://github.com/GeopJr/Tuba/blob/3ddccee4aa62b4013f7e2609174ce221d3723982/src/Dialogs/Composer/AttachmentsPage.vala#L99

**Comment by 240-185** [comment]
Got the same problem here, with a Steam Deck using Ubuntu MATE 23.10. The "Open" dialog tries to open, then nopes out. It works 10% of the time, at best. And I can't add an ALT-text.

Clicking on my profile, then the pencil merely crashes the app.

os: GNOME 45 (Flatpak runtime)
prefix: /app
flatpak: true
version: 0.5.0 (production)
gtk: 4.12.3 (4.12.1)
libadwaita: 1.4.0 (1.4.0)
libsoup: 3.4.3 (3.4.3)
libgtksourceview: 5.10.0 (5.10.0)

**Comment by GeopJr** [comment]
I don't think this is the same issue, This issue is about touch input, specifically the click and hold gesture being activated on single click. Yours sounds closer to #414, make sure your system is up to date, mostly the desktop portals!

I'll closer into it when in a vm

**Comment by GeopJr** [comment]
fixed by: #623 
The new composer is more streamlined, shouldn't be an issue.
